boards/shields/waveshare_epaper/waveshare_epaper_gdew027w3.overlay

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
possibly split the shield in another commit

Reply:
@parthitce  splitted shield related files in another commit